source/jormungandr/tests/bragi_autocomplete_tests.py

```diff
@@ -118,8 +124,13 @@ def test_autocomplete_call_override(self):
         """"
         test that the _autocomplete param switch the right autocomplete service
         """
+        url = 'https://host_of_bragi/autocomplete'
+        kwargs = {'params': {u'q': u'bob', u'type[]': [u"['stop_area', 'address', 'poi', 'administrative_region']"],
```

Comment:
I'll vote for the conversion before sending to bragi and a warning section added to the response if asked for stop_points. It's a bit like silently removing the stoppoints but at the the caller can have a feedback. If we error on this we might break integrations :confounded: